lib/core/include/scipp/core/array_to_string.h

```diff
@@ -85,7 +85,7 @@ std::string array_to_string(const T &arr,
     }
     s += element_to_string(arr[i], unit);
   }
-  s.resize(s.size() - 2);
+  s.resize(s.size() > 1 ? s.size() - 2 : 1);
```

Comment:
Should be unrelated to this PR but for some reason this started raising an exception in `concepts.ipynb`. Curiously  though, only in Jupyter. The same code in a script executed with `python` didn't raise.